server.go.tmpl

```diff
@@ -27,6 +27,14 @@ func New{{ .Name | firstLetterToUpper }}Server(svc {{$typePrefix}}{{.Name}}) Web
 }
 
 func (s *{{$serviceName}}) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	defer func() {
+		// In case of a panic, serve a HTTP 500 error and then panic.
+		if rr := recover(); rr != nil {
+			RespondWithError(w, ErrorWithCause(ErrWebrpcServerPanic, fmt.Errorf("%v", rr)))
+			panic(rr)
+		}
+	}()
```

Comment:
Move the defer-panic up to the ServeHTTP() method.

No need to define defer-panic in JSON handler for each generated RPC method, since the generated Go methods are private and called only from within `.ServerHTTP()`.